substra/sdk/client.py

```diff
@@ -110,7 +106,7 @@ def _get_backend(self, debug: bool):
             backend = backends.get(
                 "remote",
                 url=self._url,
-                version=self._version,
+                version='0.0',
```

Comment:
I think you should remove the version arg of the backends.get function, and hardcode the 0.0 in the rest_client directly.